modules/data.atmosphere/R/met2CF.Geostreams.R

```diff
@@ -53,12 +53,12 @@ met2CF.Geostreams <- function(in.path, in.prefix, outfolder,
 
     if (length(unique(dat$geometry.coordinates)) == 1) {
       # all lat/lons are are identical-- no need to store extra copies
-      raw_lat <- dat$geometry.coordinates[[1]][[1]]
```

Comment:
Just making sure, what version of clowder is this tested with? We had a bug that was fixed in 1.3.3 in which lat/lon was reversed.

Reply:
I don't know which version I'm testing against -- is there a version endpoint I can check? -- but I can confirm that the files I downloaded today use [lon, lat] everywhere whereas I just deleted some files from last July that used [lon, lat] in their sensor info followed by datapoints in [lat, lon]. So yes, this is changing to the newer standard.

Reply:
@robkooper we are using geostreams in the terraref database (1.x#420 branch:develop sha1:e91643f).